src/main/java/com/minecolonies/colony/buildings/AbstractBuilding.java

```diff
@@ -100,45 +106,51 @@ protected AbstractBuilding(Colony colony, BlockPos pos)
      *
-     * @param name              name of building.
-     * @param buildingClass     subclass of AbstractBuilding, located in {@link com.minecolonies.colony.buildings}
-     * @param parentBlock       subclass of Block, located in {@link com.minecolonies.blocks}
+     * @param name
+     *            name of building.
+     * @param buildingClass
+     *            subclass of AbstractBuilding, located in
+     *            {@link com.minecolonies.colony.buildings}
+     * @param parentBlock
+     *            subclass of Block, located in {@link com.minecolonies.blocks}
      */
-    private static void addMapping(String name, Class<? extends AbstractBuilding> buildingClass, Class<? extends AbstractBlockHut> parentBlock)
+    private static void addMapping(String name, Class<? extends AbstractBuilding> buildingClass,
+            Class<? extends AbstractBlockHut> parentBlock)
     {
         if (nameToClassMap.containsKey(name) || classNameHashToClassMap.containsKey(buildingClass.getName().hashCode()))
         {
-            throw new IllegalArgumentException("Duplicate type '" + name + "' when adding AbstractBuilding class mapping");
-        }
-        else
+            throw new IllegalArgumentException("Duplicate type '" + name
+                    + "' when adding AbstractBuilding class mapping");
+        } else
         {
             try
             {
                 /*
-                If a constructor exist for the building, put the building in the lists.
+                 * If a constructor exist for the building, put the building in
+                 * the lists.
                  */
                 if (buildingClass.getDeclaredConstructor(Colony.class, BlockPos.class) != null)
                 {
                     nameToClassMap.put(name, buildingClass);
                     classToNameMap.put(buildingClass, name);
                     classNameHashToClassMap.put(buildingClass.getName().hashCode(), buildingClass);
                 }
-            }
-            catch (NoSuchMethodException exception)
+            } catch (NoSuchMethodException exception)
             {
-                throw new IllegalArgumentException("Missing constructor for type '" + name + "' when adding AbstractBuilding class mapping", exception);
+                throw new IllegalArgumentException("Missing constructor for type '" + name
+                        + "' when adding AbstractBuilding class mapping", exception);
             }
         }
 
         if (blockClassToBuildingClassMap.containsKey(parentBlock))
         {
-            throw new IllegalArgumentException("AbstractBuilding type '" + name + "' uses TileEntity '" + parentBlock.getClass().getName() + "' which is already in use.");
-        }
-        else
+            throw new IllegalArgumentException("AbstractBuilding type '" + name + "' uses TileEntity '"
+                    + parentBlock.getClass().getName() + "' which is already in use.");
+        } else
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this "else" keyword to a new dedicated line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ARightCurlyBraceDifferentLineAsNextBlockCheck)